classes/genre.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'music_album'
+
+class Genre
+  def initialize(name, id = Random.rand(1...1000))
+    @id = id
+    @name = name
+    @items = []
+  end
+
+  def add_item(album)
+    @items << album
+  end
```

Comment:
- [ ] Please know that you need to specify the correct 1-to-many-relationship here, this is an example of how it should be: 
```ruby
    def add_item(item)
      @items.push(item)
      item.genre = self
    end
```